src/main.ts

```diff
@@ -204,6 +205,13 @@ export const fromEither = <L, A>(e: Either<L, A>) => {
   return success<L, A>(e.value);
 };
 
+export const fromOption = <L, A>(errorValue: L, o: Option<A>) => {
```

Comment:
```suggestion
export const fromOption = <L, A>(errorValue: L) => (o: Option<A>) => {
```

Pour rester dans la même philo de signatures que fp-ts

Reply:
C'est plus dans la 2.0 ce format la, je suis plus d'avis de faire une PR spécifique mise a jour 2.0

Reply:
Bah dans ce que j'utilise sur le desk, le `fromNullable` a déjà ce format là par exemple

Reply:
dans le desk c'est pas du 2.0 ?
Sinon peut faire ca dans l'autre sens

Reply:
je crois pas qu'on soit en 2.0, mais à vérifier